src/Jackett.Common/Definitions/hdtime.yml

```diff
@@ -168,8 +182,8 @@ search:
         img.pro_2up: 2
         "*": 1
     minimumratio:
-      text: 0.8
+      text: 1.0
```

Comment:
What is this based on? Previously it was the max peasant MR - https://hdtime.org/faq.php#id23